tests/foreman/virtwho/utils.py

```diff
@@ -303,3 +307,48 @@ def deploy_configure_by_script(script_content, debug=False):
         )
     if debug:
         return deploy_validation()
+
+
+def restart_virt_who_service():
+    """
+    Do the following:
+    1. clean rhsm.log message, make sure there is no old message exist.
+    2. restart virt-who service via systemctl command
+    """
+    runcmd("rm -f /var/log/rhsm/rhsm.log")
+    runcmd("systemctl restart virt-who; sleep 5")
+
+
+def update_configure_option(option, value, config_id):
```

Comment:
It's more better if use config_file as the argument, thus, this function can be used to update other config_file as well, such as update /etc/sysconfig/virt-who.

Reply:
thanks, modified.